### PR TITLE
IGNITE-18063 Use consistent IDs in assignments

### PR DESCRIPTION
**Pull request by sashapolo**
https://issues.apache.org/jira/browse/IGNITE-18063

### CODE REVIEW

File: modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/rel/Outbox.java [hunk 1]
```diff
@@ -95,15 +95,15 @@ public long exchangeId() {
     /**
      * Callback method.
      *
-     * @param nodeId  Target ID.
+     * @param nodeName  Target consistent ID.
```
Comment (ibessonov) [inline modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/rel/Outbox.java:98]:
```suggestion
     * @param nodeName Target consistent ID.
```

File: modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/metadata/RemoteException.java [hunk 2]
```diff
@@ -34,22 +34,22 @@ public class RemoteException extends RuntimeException {
      * Constructor.
      *
      * @param cause      Cause.
-     * @param nodeId     Node ID.
+     * @param nodeName     Node consistent ID.
```
Comment (ibessonov) [inline modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/metadata/RemoteException.java:37]:
```suggestion
     * @param nodeName Node consistent ID.
```

Comment (ibessonov) [reply modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/metadata/RemoteException.java:37]: Please format other lines too

**Review by korlov42 — APPROVED** [review]

**Review by SammyVimes — APPROVED** [review]
LGTM

**Pull request merged**